src/applications/edu-benefits/10203/containers/ConfirmEligibilityView.jsx

```diff
@@ -107,125 +103,69 @@ export class ConfirmEligibilityView extends React.Component {
+                      <strong>
+                        {this.props.remainingEntitlement.months} months,{' '}
+                        {this.props.remainingEntitlement.days} days{' '}
+                      </strong>
+                    </p>
                   </div>
                 </div>
-                <br />
               </div>
-            )}
-          <span>
-            If your situation changes in the future and you meet all of the
-            criteria, you may return to apply for the Rogers STEM Scholarship.
-          </span>
-          <div className="vads-u-padding-top--2">
-            <a
-              className={'usa-button-primary wizard-button va-button-primary'}
-              href="/education/about-gi-bill-benefits/"
-            >
-              Explore other education benefits
-            </a>
-          </div>
-        </div>
-
-        <div>
-          <fieldset className="schemaform-field-template schemaform-first-field">
-            <legend className={legendClassName}>
-              {questionText}
-              <span className="schemaform-required-span">(*Required)</span>
-            </legend>
-            {this.renderErrorMessage()}
-            <input
-              type="radio"
-              checked={confirmEligibility === false}
-              id={`${id}No`}
-              name={`${id}`}
-              value="N"
-              onChange={() =>
-                this.onChange({ 'view:confirmEligibility': false })
-              }
-              onClick={() => captureEvents.ineligibilityStillApply(false)}
-            />
-            <label htmlFor={`${id}No`}>No</label>
-            <input
-              type="radio"
-              checked={confirmEligibility === true}
-              id={`${id}Yes`}
-              name={`${id}`}
-              value="Y"
-              onChange={() =>
-                this.onChange({ 'view:confirmEligibility': true })
-              }
-              onClick={() => {
-                captureEvents.ineligibilityStillApply(true);
-              }}
-            />
-            <label htmlFor={`${id}Yes`}>Yes</label>
-          </fieldset>
-        </div>
+            </div>
+          )}
       </div>
     );
   };
 
   render() {
     return (
       <div>
+        <div className="vads-u-padding-bottom--2">
+          <h4>Rogers STEM Scholarship eligibility summary</h4>
+        </div>
+
         <div className="vads-u-background-color--gray-lightest vads-u-padding-y--1 vads-u-padding-x--2">
           {this.renderChecks()}
           {this.renderConfirmEligibility()}
         </div>
+        <div className="vads-u-padding-y--4">
+          <b>
+            You must meet the above requirements to qualify for the scholarship.
+          </b>{' '}
+          Please consider that ineligible applications delay the processing of
+          benefits for eligible applicants.
+        </div>
+        <div className="vads-u-margin-top--neg2">
```

Comment:
i would hide the "exit" button on the review page with `this.props.onReviewPage` because it won't make sense to have the button display on review page

Reply:
Good catch

Reply:
Button hidden on review page